polyfill/spatial-navigation-polyfill.js

```diff
@@ -92,7 +92,10 @@
       if (!currentKeyMode ||
           (currentKeyMode === 'NONE') ||
           ((currentKeyMode === 'SHIFTARROW') && !e.shiftKey) ||
-          ((currentKeyMode === 'ARROW') && e.shiftKey))
+          ((currentKeyMode === 'ARROW') && e.shiftKey) ||
+          e.ctrlKey ||
```

Comment:
How about (e.ctrlKey || e.metaKey || e.altKey)) as a single expression for a purpose excluding the unrelated modifier keys?

Reply:
good idea